docs/getting-started/who-supports-saf.md

```diff
@@ -44,9 +44,9 @@ See below overview tables which will give you an idea of who supports SAF and to
 | RelConnectsRigidLink        |                                 Yes                                 |                                   No                                  |                                           Yes                                           |                                   No                                  |                                       No                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                          |                                          No                                          | Yes | No | No | No |
-| RelConnectsRigidMember      |                                  No                                 |                                   No                                  |                                           Yes                                           |                                   No                                  |                                       No                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                          |                                          No                                          | Yes | No | No | No |
+| RelConnectsRigidMember      |                                  Yes                                 |                                   No                                  |                                           Yes                                           |                                   No                                  |                                       No                                      |                                           No                                           |                                   Yes                                   |                                           Yes                                          |                                          No                                          | Yes | No | No | No |
```

Comment:
These changes are not described in the issue https://github.com/StructuralAnalysisFormat/StructuralAnalysisFormat-Doc/issues/138, I've noticed there are obsolete information fro SCIA, so fixed them here.